test/lensEq.js

```diff
@@ -0,0 +1,57 @@
+var R = require('../source');
+var eq = require('./shared/eq');
+
+
+describe('lensEq', function() {
+  var obj = {name: 'Abby', age: 7, hair: 'blond', family: { father: 'Jones' }};
+  var array = [1, 2, 3, 4];
+
+  it('able to use with lens', function() {
```

Comment:
Two things here:

  - I don't think there's any reason to call out tests for different implementations of lenses.  Lenses are their own abstraction which can be tested in their own implementations (`lens`, `lensIndex`, `lensPath`, etc.)  While it would probably be a good idea if the various test cases used a few different lens creators, there is no particular reason to call attention to it in the test descriptions.

  - in this pseudo-BDD style of tests, I really prefer if the the description read as sentences when appended to the `describe` value.  That is

    ```
    describe ('My noun', () => {
      it ('has properties described by this phrase')
    })
     ```

    which of course is read in the output as "My noun has properties described by this phrase."